templates/taxi/car_list.html

```diff
@@ -2,15 +2,50 @@
 
 {% block content %}
   <h1>Car list</h1>
+  <h1>Car list
+    <a style="float: right" href="{% url "taxi:car-create" %}">+</a>
+  </h1>
   {% if car_list %}
     <ul>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+
       {% for car in car_list %}
         <li>
           <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
+        <tr>
+          <td>
+            {{ car.id }}
+          </td>
+          <td>
+            {{ car.model }}
+          </td>
+          <td>
+            {{ car.manufacturer.name }}
+          </td>
+          <td>
+            <a href="{% url "taxi:car-update" pk=car.id %}">
+              UPDATE
+            </a>
+          </td>
+          <td>
+            <a href="{% url "taxi:car-delete" pk=car.id %}">
+              DELETE
+            </a>
+          </td>
+        </tr>
       {% endfor %}
     </ul>
+    </table>
```

Comment:
The `<ul>` tag on line 9 is unnecessary since the car list is being displayed in a table format. Consider removing the `<ul>` tag to clean up the HTML structure.